students/jim_krecek/lesson02/series.py

```diff
@@ -0,0 +1,66 @@
+def fibonacci(n):
+	""" compute the nth Fibonacci number """
+	if n == 0:
+		return n
+	elif n == 1:
+		return n
+	else:
+		return (fibonacci(n-2) + fibonacci(n-1))
+			
+def lucas(n):
+	""" compute the nth Lucas number """
+	if n == 0:
+		return 2
+	elif n == 1:
+		return n
+	else:
+		return (lucas(n-2) + lucas(n-1))
+
+def sum_series(n, x=0, y=1):
+	"""
+	compute the nth value of a summation series.
+
+	:param n0=0: value of zeroth element in the series
+	:param n1=1: value of first element in the series
+
+	This function should generalize the fibonacci() and the lucas(),
+	so that this function works for any first two numbers for a sum series.
+
+	Once generalized that way, sum_series(n, 0, 1) should be equivalent to fibonacci(n).
+	And sum_series(n, 2, 1) should be equivalent to lucas(n).
+	"""
+	if n == 0:
+		return x
+	elif n == 1:
+		return y
+	else:
+		return (sum_series(n-2,x,y) + sum_series(n-1,x,y))
```

Comment:
no need for parens with `return` statement as it isn't a function -> `return sum_series(n-2, x, y) + sum_series(n-1, x, y)`